lib/Basics/AssocMulti.h

```diff
@@ -248,7 +248,7 @@ class AssocMulti {
     // index, i.e. when the index is built for a collection and we know
     // for sure no duplicate elements will be inserted
 
-#ifdef TRI_CHECK_MULTI_POINTER_HASH
+#ifdef ARANGODB_CHECK_MULTI_POINTER_HASH
```

Comment:
why not ADB_* `

Reply:
I think we should use the product/company name in its full glory here, just to avoid (unlikely, but possible) overlaps with third-party defines.
The long-term plan should be to rename all TRI_ and ADB_ things to ARANGODB_ as well.
This isn't done in this PR to avoid merge conflicts in some of our big pending PRs (hot backup, ApplicationServer refactoring, one shard). But we should do this afterwards, so the naming gets consistent.